research-software-directory/RSD-as-a-service
# 448 delete software, project, organisation, community from RSD

# Admin can delete software,project,organisation & community from RSD

Closes #488

Changes proposed in this pull request:
* RSD admin can delete software, project, organisation and community from admin page
* We also try to remove images (logo/project image) from RSD
* Order on software is not published, not updated
* Order on projects is not published, not updated
* Click on item to open it for edit (instead of using edit button)

How to test:
* `make start` to build app and generate test data
* login as rsd admin
* navigate to admin pages
* navigate to software and try to remove a software item. Conform the item is deleted.
* navigate to project page and try to remove project. Confirm the item is removed.
* navigate to organisations page and try to remove organisation. Confirm that organisation is removed (it is also removed from software and project pages)
* navigate to community page and remove community. Confirm the community is removed

## Example delete software
![image](https://github.com/user-attachments/assets/bf706868-c353-4b03-8c04-d6eac74e3060)

## Example delete project
![image](https://github.com/user-attachments/assets/1578540f-e969-4e32-a413-a9ba29954f00)

## Example delete organisation
![image](https://github.com/user-attachments/assets/1228c158-3bcf-41f2-af1c-1bb1beef126b)

## Example delete community
![image](https://github.com/user-attachments/assets/e5c3d4a6-3cbd-4ac1-9958-3ca7d1a2e5bc)


PR Checklist:
